bin/json2xlsx.js

```diff
@@ -5,10 +5,12 @@ const run = require('../src/main/json2xlsx');
 run(parseArgs(process.argv.slice(2), {
   string: [
     'D',
-    'o'
+    'o',
+    'e'
   ],
   alias: {
     sheetsDir: 'D',
-    destinationXlsx: 'o'
+    destinationXlsx: 'o',
+    exclude: 'e'
```

Comment:
Maybe rename to `excludedFiles`

Reply:
excludedFilenamePatterns as excludedFiles is not 100% true